articles/quickstart/backend/golang/interactive.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Add authorization to a Go API
+description: This tutorial demonstrates how to add authorization to a Go API.
+topics:
+  - quickstart
+  - backend
+  - golang
+interactive: true
+github:
+  path: 01-Authorization-RS256
+contentType: tutorial
+useCase: quickstart
+files:
+  - files/jwt
+  - files/main
+---
+
+<!-- markdownlint-disable MD041 MD025 -->
+
+# Add Authorization to a Go application.
```

Comment:
Style guide: titles should use sentence case unless it's referring to an Auth0 product or proper noun

```suggestion
# Add authorization to a Go application.
```